nsv13/code/modules/overmap/radar.dm

```diff
@@ -1,8 +1,35 @@
+/obj/item/circuitboard/computer/ship/dradis
+	name = "circuit board (dradis computer)"
+	build_path = /obj/machinery/computer/ship/dradis
+
+/datum/design/board/dradis_circuit
```

Comment:
See how the freezer and heater board is done for the atmos machines and do the code needed to let the screwdriver switching between this one and the child mining one.